### PR TITLE
Release notes for set streaming. Bump minor version.

### PR DESCRIPTION
**Pull request by doom-weaver**
###

Change log and version bump for set streaming.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-## 4.1.1 [current]
+## 4.2.0 [current]
+- Add support for [set streaming](https://docs.fauna.com/fauna/current/drivers/streaming).
```
Comment (elersong) [inline CHANGELOG.md]: Can you add a link to the PR?
`Add support for set streaming [###](link to pr)`

**Review by elersong — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (doom-weaver) [reply CHANGELOG.md]: this will be customer facing - figured the docs would be more valuable.

Comment (faunaee) [reply CHANGELOG.md]: The documentation link is a good idea, but I agree with Grey that the PR/commit providing the feature should be included.

**Review by faunaee — APPROVED** [review]
With Grey's comment addressed, LGTM! 👍

File: CHANGELOG.md (hunk 1 above)
Comment (doom-weaver) [reply CHANGELOG.md]: 👍

Comment (doom-weaver) [inline CHANGELOG.md]:
```suggestion
- Add support for [set streaming](https://docs.fauna.com/fauna/current/drivers/streaming). Pull request for set streaming [here](https://github.com/fauna/faunadb-python/pull/221).
```

**Review by elersong — APPROVED** [review]

**Pull request merged**